src/styles/blocks/stars.scss

```diff
@@ -5,11 +5,11 @@
 
   &__item {
     background-image: url(\images\star-active.svg);
-    background-position: center;
-    background-repeat: no-repeat;
     height: 15px;
     width: 15px;
     margin-right: 4px;
+    background-position: center;
```

Comment:
where is css changes, that handle modifier, that you've added to the stars block?